[2025-10-07T00:00:42.780+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: pytorch_circle_classifier.test_model scheduled__2025-10-06T00:00:00+00:00 [queued]>
[2025-10-07T00:00:42.790+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: pytorch_circle_classifier.test_model scheduled__2025-10-06T00:00:00+00:00 [queued]>
[2025-10-07T00:00:42.790+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-10-07T00:00:42.790+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-10-07T00:00:42.790+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-10-07T00:00:42.798+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): test_model> on 2025-10-06 00:00:00+00:00
[2025-10-07T00:00:42.807+0000] {standard_task_runner.py:55} INFO - Started process 101 to run task
[2025-10-07T00:00:42.810+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pytorch_circle_classifier', 'test_model', 'scheduled__2025-10-06T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpmue0omq4']
[2025-10-07T00:00:42.812+0000] {standard_task_runner.py:83} INFO - Job 21: Subtask test_model
[2025-10-07T00:00:42.848+0000] {task_command.py:388} INFO - Running <TaskInstance: pytorch_circle_classifier.test_model scheduled__2025-10-06T00:00:00+00:00 [running]> on host 1c9de8940ef2
[2025-10-07T00:00:43.481+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Aditya Ratan
AIRFLOW_CTX_DAG_ID=pytorch_circle_classifier
AIRFLOW_CTX_TASK_ID=test_model
AIRFLOW_CTX_EXECUTION_DATE=2025-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-10-06T00:00:00+00:00
[2025-10-07T00:00:43.482+0000] {logging_mixin.py:137} INFO - Testing model with new data...
[2025-10-07T00:00:43.488+0000] {logging_mixin.py:137} INFO - Model loaded from ./models/circle_classifier.pth
[2025-10-07T00:00:43.488+0000] {logging_mixin.py:137} INFO - Training accuracy was: 1.0000
[2025-10-07T00:00:43.489+0000] {logging_mixin.py:137} INFO - Test input: x1=0.01, x2=0.2
[2025-10-07T00:00:43.492+0000] {logging_mixin.py:137} INFO - 
--- Prediction Results ---
[2025-10-07T00:00:43.492+0000] {logging_mixin.py:137} INFO - Input: (0.0100, 0.2000)
[2025-10-07T00:00:43.492+0000] {logging_mixin.py:137} INFO - Probability: 0.9997
[2025-10-07T00:00:43.492+0000] {logging_mixin.py:137} INFO - Predicted Label: 1
[2025-10-07T00:00:43.492+0000] {logging_mixin.py:137} INFO - Classification: Inside circle
[2025-10-07T00:00:43.492+0000] {logging_mixin.py:137} INFO - Results saved to ./results/prediction_result.json
[2025-10-07T00:00:43.492+0000] {python.py:177} INFO - Done. Returned value was: {'x1': 0.01, 'x2': 0.2, 'probability': 0.999692440032959, 'predicted_label': 1, 'label_description': 'Inside circle', 'true_label': 'Outside circle'}
[2025-10-07T00:00:43.512+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=pytorch_circle_classifier, task_id=test_model, execution_date=20251006T000000, start_date=20251007T000042, end_date=20251007T000043
[2025-10-07T00:00:43.536+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-10-07T00:00:43.566+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
